extensions/obsidian-bookmarks/package.json

```
@@ -145,6 +148,15 @@
       "type": "textfield",
       "placeholder": "home, work, fun",
       "required": false
+    },
+    {
+      "name": "datePrefix",
+      "label": "Date Prefix",
+      "title": "Date Prefix",
+      "description": "The prefix to use for the date in the filename of your bookmarks.",
+      "type": "checkbox",
+      "default": true,
+      "required": true
```

```suggestion
      "default": false,
      "required": false
```
Would be better to set it like this so we don't break users flow with a new preference

---

I set the default to true because in previous versions it would automatically add the time as a prefix.

And I think setting require to false is good, and I will fix it promptly.

---

Ah, that makes sense, my eyes didn't catch that 🙂